app/services/affiliate-slots-units.json

```diff
@@ -26,7 +26,7 @@
     "name": "xkxd02",
     "isBig": true,
     "image": "https://vignette.wikia.nocookie.net/central/images/b/bc/Wiki.png/revision/latest?cb=20180423162614",
-    "heading": "Ewan McGregor Returns as Obi-Wan Kenobi in Disney+ Series",
+    "heading": "BIG Ewan McGregor Returns as Obi-Wan Kenobi in Disney+ Series",
```

Comment:
?

Reply:
It's just for testing. I wanted to have a distinction between BIG unit and small unit.